config/initializers/paperclip.rb

```diff
@@ -0,0 +1,14 @@
+Paperclip::Attachment.default_options.update({
+  s3_host_name: "s3-eu-west-1.amazonaws.com",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.